lib/token.h

```diff
@@ -63,6 +63,15 @@ struct ScopeInfo2 {
 
 enum class TokenDebug : std::uint8_t { None, ValueFlow, ValueType };
 
+// TODO: move these to Token when TokenImpl has been moved into the implementation
+enum CppcheckAttributesType : std::uint8_t { LOW, HIGH };
+struct CppcheckAttributes {
+    CppcheckAttributesType type{LOW};
+    MathLib::bigint value{};
+    CppcheckAttributes* next{};
+};
```

Comment:
I split the type out since that's the only part which is actually used in `Token`.